src/plugins/stake-plugins/thorchainSavers/tcSavers.ts

```diff
@@ -2,13 +2,14 @@ import { add, div, lt, mul, sub, toFixed } from 'biggystring'
 import { asArray, asBoolean, asNumber, asObject, asOptional, asString } from 'cleaners'
 import { EdgeCorePluginOptions, EdgeCurrencyWallet, InsufficientFundsError } from 'edge-core-js'
 
+import { showError } from '../../../components/services/AirshipInstance'
```

Comment:
Can do that for fetchStakePosition since an error means we really need to throw. However for makeTcSaversPlugin, need to find a better solution since we can't throw as that would kill the whole plugin even though the errors would only prevent APY display.